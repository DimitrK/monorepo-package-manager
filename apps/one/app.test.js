import React from 'react';
import {render} from '@testing-library/react';
import App from './app'

describe('app', () => {
  test('matches snapshot', () => {
    const {container} = render(<App/>);
    expect(container).toMatchSnapshot();
  })
})