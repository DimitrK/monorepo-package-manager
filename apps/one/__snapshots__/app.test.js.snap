// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app matches snapshot 1`] = `
<div>
  <div>
    <p>
      Hello from app One
    </p>
    <button>
      I am a shared component
    </button>
  </div>
</div>
`;
